src/tmx/TmxCore/CMakeFiles/tmxcore.dir/CXX.includecache

```diff
@@ -0,0 +1,880 @@
+#IncludeRegexLine: ^[ 	]*[#%][ 	]*(include|import)[ 	]*[<"]([^">]+)([">])
+
+#IncludeRegexScan: ^.*$
+
+#IncludeRegexComplain: ^$
+
+#IncludeRegexTransform: 
+
+/home/anjanrayamajhi/Projects/V2X-CARMACloud/V2X-Hub/src/tmx/TmxCore/src/HistoryManager.cpp
```

Comment:
This file contains specific username home path. Can we remove this cache file? or git ignore it?